tests/rptest/tests/shadow_indexing_tx_test.py

```diff
@@ -28,46 +28,26 @@
 
 class ShadowIndexingTxTest(RedpandaTest):
     segment_size = 1048576  # 1 Mb
-    s3_host_name = "minio-s3"
-    s3_access_key = "panda-user"
-    s3_secret_key = "panda-secret"
-    s3_region = "panda-region"
-    s3_topic_name = "panda-topic"
     topics = (TopicSpec(name='panda-topic',
                         partition_count=1,
                         replication_factor=3), )
 
     def __init__(self, test_context):
         self.s3_bucket_name = f"panda-bucket-{uuid.uuid1()}"
```

Comment:
SISettings auto-gens the bucket name so is this line needed?

Reply:
Yep, this became dead code.  Removed.